collectors/slurm/src/sacctcaller.rs

```diff
@@ -95,23 +95,30 @@ async fn place_records_on_queue(records: Vec<RecordAdd>, tx: &mpsc::Sender<Recor
 async fn get_job_info(database: &Database) -> Result<Vec<RecordAdd>> {
     let (lastcheck, last_record_id) = database.get_lastcheck().await?;
 
-    let cmd_out = Command::new("/usr/bin/sacct")
-        .arg("-a")
-        .arg("--format")
-        .arg(KEYS.iter().map(|k| k.0.clone()).join(","))
-        .arg("--noconvert")
-        .arg("--noheader")
-        .arg("-S")
-        .arg(format!("{}", lastcheck.format("%Y-%m-%dT%H:%M:%S")))
-        .arg("-E")
-        .arg("now")
-        .arg("-s")
-        .arg(CONFIG.job_status.join(","))
-        .arg("-P")
-        .output()
-        .await?;
+    let binary = "/usr/bin/sacct";
+    let mut args: Vec<String> = Vec::new();
+    args.push("-a".to_string());
+    args.push("--format".to_string());
+    args.push(KEYS.iter().map(|k| k.0.clone()).join(","));
+    args.push("--noconvert".to_string());
+    args.push("--noheader".to_string());
+    args.push("-S".to_string());
+    args.push(format!("{}", lastcheck.format("%Y-%m-%dT%H:%M:%S")));
+    args.push("-E".to_string());
+    args.push("now".to_string());
+    args.push("-s".to_string());
+    args.push(CONFIG.job_status.join(","));
+    args.push("-P".to_string());
```

Comment:
This isn't good style, I would use the vec![] macro instead.

Reply:
Thanks for the hint! This was just a quick hack to get some debug output. I needed the PR to let GH Actions build the RPMs, as I couldn't get the compilation to the musl target working. I fixed this now by using a `vec![]` macro.